modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/yaml/graphql/GraphQLNamingUtil.java

```diff
@@ -1,50 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.portal.vulcan.yaml.graphql;
-
-import com.liferay.portal.kernel.util.StringUtil;
-
-import java.util.List;
-
-/**
- * @author     Javier Gamarra
- * @deprecated As of Athanasius (7.3.x), replaced by {@link
- *             com.liferay.portal.vulcan.graphql.util.GraphQLNamingUtil}
- */
-@Deprecated
-public class GraphQLNamingUtil {
```

Comment:
Why are you removing this? I think this might be used by customers. I am not sure if removing this interface inside a story that handles the siteKey is the best idea